shoop_tests/xtheme/test_addon_injections.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from shoop.apps.provides import override_provides
+from shoop.xtheme.theme import override_current_theme_class
+from shoop.xtheme.resources import InlineScriptResource, add_resource
+from shoop_tests.xtheme.utils import get_jinja2_engine, get_request
+
+
+def add_test_injection(context):
+    add_resource(context, "body_end", InlineScriptResource("window.injectedFromAddon=true;"))
+
+
+@pytest.mark.django_db
+def test_simple_addon_injection():
+    request = get_request(edit=False)
+    jeng = get_jinja2_engine()
+    template = jeng.get_template("resinject.jinja")
+
+    # By default window.injectedFromAddon=true; should not be in resinject.jinja body
+    output = template.render(request=request)
+    head, body = output.split("</head>", 1)
```

Comment:
You don't need to do this splitting thing in this test, I don't think -- we have other tests to check that `body_end` works as it should.